go/core.rst

```diff
@@ -0,0 +1,425 @@
+| :param:`embed`             | :type:`label_list`          | :value:`None`                         |
++----------------------------+-----------------------------+---------------------------------------+
+| List of GoEmbed_ providers this library embeds.                                                  |
+| These can provide both :param:`srcs` and param:`deps` to this library.                           |
+| See Embedding_ for more information about how and when to use this.                              |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`data`              | :type:`label_list`          | :value:`None`                         |
++----------------------------+-----------------------------+---------------------------------------+
+| The list of files needed by this rule at runtime. Targets named in the data attribute will       |
+| appear in the *.runfiles area of this rule, if it has one. This may include data files needed    |
+| by the binary, or other programs needed by it. See `data dependencies`_ for more information     |
+| about how to depend on and use data files.                                                       |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`gc_goopts`         | :type:`string_list`         | :value:`[]`                           |
++----------------------------+-----------------------------+---------------------------------------+
+| List of flags to add to the Go compilation command when using the gc compiler.                   |
+| Subject to `"Make variable"`_ substitution and `Bourne shell tokenization`_.                     |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`cgo`               | :type:`boolean`             | :value:`False`                        |
++----------------------------+-----------------------------+---------------------------------------+
+| If :value:`True`, the package uses cgo_.                                                         |
+| The cgo tool permits Go code to call C code and vice-versa.                                      |
+| This does not support calling C++.                                                               |
+| When cgo is set, :param:`srcs` may contain C or assembly files; these files are compiled with    |
+| the normal c compiler and included in the package.                                               |
++----------------------------+-----------------------------+---------------------------------------+
+| :param:`cdeps`             | :type:`label_list`          | :value:`None`                         |
++----------------------------+-----------------------------+---------------------------------------+
+| The list of other libraries that the c code depends on.                                          |
```

Comment:
List of `cc_library` rules the cgo code depends on. Only valid if `cgo = True`.

(Though I don't think we actually check that).

Reply:
Done